org-formation/600-access/_tasks.yaml

```diff
@@ -304,6 +304,8 @@ SynapseAthenaUserAccessPolicy:
                 "Resource": [
                     "arn:aws:s3:::aws-athena-query-results-${CurrentAccount.AccountId}-us-east-1",
                     "arn:aws:s3:::aws-athena-query-results-${CurrentAccount.AccountId}-us-east-1/*",
+                    "arn:aws:s3:::aws-athena-query-results-us-east-1-${CurrentAccount.AccountId}",
```

Comment:
the two ARNs just above are also athena-query-results buckets.  Do these new ARNs replace the existing ARNs?

Reply:
No, the new ARN doesn't replace the current ARN but rather complements it.  Different tools (Athena and Quicksight) seem to require different bucket names.